client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -1570,12 +1570,12 @@ protected ReduceFileGroups loadFileGroupInternal(int shuffleId) {
     }
   }
 
-  protected ReduceFileGroups updateFileGroup(int shuffleId, int partitionId) throws IOException {
+  protected ReduceFileGroups updateFileGroup(int shuffleId) {
```

Comment:
`partitionId` is used in `FlinkShuffleClientImpl`

Reply:
sorry，I will revert this change